AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/consts/scripting.py

```diff
@@ -0,0 +1,60 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Holds script canvas related constants
+"""
+import os
+import azlmbr.paths as paths
+
+"""
+Constants for window and tab names
+
+"""
+SCRIPT_CANVAS_UI = "Script Canvas"
```

Comment:
If the tests are included in AR, yes. Otherwise they'd run in the overnight and trigger then.

@michleza we should create a backlog item to talk to the UI team to get the UI exposed to python through PYI files so we can replace these UI strings with an azlmbr call. This is what @Kadino was discussing with us in DM about last week.

Reply:
this will be something to come back to in the future. we need engineering support to retrieve label strings used in testing.